standard/standard_1.php

```diff
@@ -418,7 +418,11 @@ function str_split(string $string, int $length = 1): array|false {}
  * not found.
  */
 #[Pure]
-function strpbrk(string $string, string $characters): string|false {}
+function strpbrk(
```

Comment:
https://3v4l.org/RW51p#v540:
1) till 7.4 `$string` called `$haystack` 
2) `$char_list` had no default value

Reply:
1. we don't care about param names prior 8.0 so it's fine
2. yep, but still it was optional so we have to put default value to make it optional